web/components/VenueNames/VenueNames.module.css

```diff
@@ -4,6 +4,20 @@
   padding: 0;
 }
 
+.venuesEnter .venue {
```

Comment:
Nit: equivalent comments to the above on `.navEnter .item`, regarding position and class names. (I.e. I'd move this below the `.venue` ruleset, and maybe do something like `.venues.isIntersecting .venue` instead. Looks like the `.venue:nth-child(n + 2)::before` ruleset is already breaking the convention…)